iamproperty/iamproperty.github.io
# Accordion

## Summary of Changes:
1. Accordion docs
2. Accordion component
3. Accordion CSS
4. Unit tests

## Rationale
Create a component to be used on pages like FAQs to group a list of questions that can be expanded to show the answer.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/accordion

## Ticket(s)
DS-64

## Checklist

## Notes
